test/opexecuter.cpp

```diff
@@ -127,7 +130,12 @@ OpExecuter::~OpExecuter() {}
 void OpExecuter::RunTest() {
   ExecuteOnNGraph();
   ExecuteOnTF();
-  CompareNGraphAndTF();
+  CompareNGraphAndTF(tf_outputs_, ngraph_outputs_);
+}
```

Comment:
It is a good idea to have CompareNGraphAndTF in the utility

however with the slight change (using getter), we don't have to change CompareNGraphAndTF's interface (which earlier was accepting nothing, but now accepts 2 inputs)